content/ember/v6/deprecate-ember-object-observers.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Deprecation of @ember/object/observers
+until: 7.0.0
+since: 6.5.0
+---
+
+The `addObserver` and `removeObserver` methods from `@ember/object/observers` are deprecated. Instead of using observers, you should use tracked properties and native getters/setters.
```

Comment:
we need another use case for before/after:
- ember-concurrency's `waitFor` uses observers

and whatever these do:
- ember-animated
- ember-data